src/demo-app/tabs/tabs-demo.ts

```diff
@@ -133,3 +133,16 @@ export class RainyTabContent {}
   template: 'This is the routed body of the foggy tab.',
 })
 export class FoggyTabContent {}
+
+@Component({
+  moduleId: module.id,
+  selector: 'counter',
+  template: `<span>{{count}}</span>`
+ })
+ export class Counter {
+  count = 0;
+  ngOnInit() {
+    this.count++;
```

Comment:
Not totally sure what the count represents, shouldn't this be 1 for any component regardless of how it is loaded?

Reply:
To be honest, Its been so long I forgot what I was using it for too. I removed it.